README.md

```diff
@@ -66,14 +66,5 @@ For more info, check out the compatibility matrix on Docker's website: [compatib
 https://docs.docker.com/compose/compose-file/compose-versioning/#compatibility-matrix)
 
 ### Building the docker image
-We expect a build-arg for which node-version to build with so if you're building using the docker cli, you have to use `--build-arg NODE_VERSION=14-alpine` OR `--build-arg NODE_VERSION=16-alpine`
 
-## Upgrade version
-When we upgrade the `unleash-version` this project should be tagged with the same version number.
-
-```bash
-git tag -a v3.7.0 -m "upgrade to unleash-server v3.7.0"
-git push origin main --follow-tags
-```
-
-This will automatically trigger a github actions which will build the new tag and push it to docker-hub. 
+The docker image is very small shim on top of unleashorg/unleash to include wait-for and allow the docker-compose configuration to ensure that we're not starting unleash until the database is actually up.
```

Comment:
Maybe we should move this paragraph to the start of the readme and remove the "Building the docker image" heading? It doesn't really describe how to build it at the moment, and this paragraph reads more like a description of the repo,  think.

Maybe something like this would clarify what the purpose of the repo is? 

```suggestion
This repo contains a docker compose file and an extended dockerfile for running the Unleash server. The docker compose file will start a database, the Unleash server, and an Unleash proxy.

The extended Unleash dockerfile is very small shim on top of [unleash/unleash](https://github.com/Unleash/unleash/) to include wait-for and allow the docker-compose configuration to ensure that we're not starting Unleash until the database is up and running.
```